### PR TITLE
[BACKLOG-38136] PUC UI changes

### PR DESCRIPTION
**Pull request by srallapa**

### CODE REVIEW

**Comment by buildguy** [comment]
### :warning: Build finished in 1h 34m
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl extensions
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 5 test(s) failed:

<details><summary><b>org.pentaho.platform.plugin.services.importexport.exportManifest.ExportManifestTest.testToXml_XmlUnsafeEscaped</b> (click to expand)</summary><p>

```
Could not marshal to XML com.sun.xml.bind.v2.runtime.IllegalAnnotationsException: 3 counts of IllegalAnnotationExceptions
Property runSafeMode is present but not specified in @XmlType.propOrder
 this problem is related to the following location:
  at protected java.lang.String org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest.runSafeMode
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest
  at public org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory.createJobScheduleRequest()
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory
  at protected org.w3c.dom.Element org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty.any
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty.customProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto.exportManifestProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto.exportManifestEntity
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto
Property gatheringMetrics is present but not specified in @XmlType.propOrder
 this problem is related to the following location:
  at protected java.lang.String org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest.gatheringMetrics
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest
  at public org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory.createJobScheduleRequest()
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory
  at protected org.w3c.dom.Element org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty.any
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty.customProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto.exportManifestProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto.exportManifestEntity
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto
Property logLevel is present but not specified in @XmlType.propOrder
 this problem is related to the following location:
  at protected java.lang.String org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest.logLevel
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest
  at public org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory.createJobScheduleRequest()
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory
  at protected org.w3c.dom.Element org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty.any
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty.customProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto.exportManifestProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto.exportManifestEntity
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto

```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.importexport.exportManifest.ExportManifestTest.testToXml_XmlUnsafeManifestParameters</b> (click to expand)</summary><p>

```
Could not marshal to XML com.sun.xml.bind.v2.runtime.IllegalAnnotationsException: 3 counts of IllegalAnnotationExceptions
Property runSafeMode is present but not specified in @XmlType.propOrder
 this problem is related to the following location:
  at protected java.lang.String org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest.runSafeMode
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest
  at public org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory.createJobScheduleRequest()
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory
  at protected org.w3c.dom.Element org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty.any
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty.customProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto.exportManifestProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto.exportManifestEntity
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto
Property gatheringMetrics is present but not specified in @XmlType.propOrder
 this problem is related to the following location:
  at protected java.lang.String org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest.gatheringMetrics
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest
  at public org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory.createJobScheduleRequest()
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory
  at protected org.w3c.dom.Element org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty.any
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty.customProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto.exportManifestProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto.exportManifestEntity
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto
Property logLevel is present but not specified in @XmlType.propOrder
 this problem is related to the following location:
  at protected java.lang.String org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest.logLevel
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest
  at public org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory.createJobScheduleRequest()
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory
  at protected org.w3c.dom.Element org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty.any
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty.customProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto.exportManifestProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto.exportManifestEntity
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto

```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.importexport.exportManifest.ExportManifestTest.testXmlToString</b> (click to expand)</summary><p>

```
Could not marshal to XML to string com.sun.xml.bind.v2.runtime.IllegalAnnotationsException: 3 counts of IllegalAnnotationExceptions
Property runSafeMode is present but not specified in @XmlType.propOrder
 this problem is related to the following location:
  at protected java.lang.String org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest.runSafeMode
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest
  at public org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory.createJobScheduleRequest()
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory
  at protected org.w3c.dom.Element org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty.any
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty.customProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto.exportManifestProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto.exportManifestEntity
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto
Property gatheringMetrics is present but not specified in @XmlType.propOrder
 this problem is related to the following location:
  at protected java.lang.String org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest.gatheringMetrics
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest
  at public org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory.createJobScheduleRequest()
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory
  at protected org.w3c.dom.Element org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty.any
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty.customProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto.exportManifestProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto.exportManifestEntity
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto
Property logLevel is present but not specified in @XmlType.propOrder
 this problem is related to the following location:
  at protected java.lang.String org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest.logLevel
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest
  at public org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory.createJobScheduleRequest()
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory
  at protected org.w3c.dom.Element org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty.any
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty.customProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto.exportManifestProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto.exportManifestEntity
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto

```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.importexport.exportManifest.ExportManifestTest.testMarshal</b> (click to expand)</summary><p>

```
Could not marshal to XML com.sun.xml.bind.v2.runtime.IllegalAnnotationsException: 3 counts of IllegalAnnotationExceptions
Property runSafeMode is present but not specified in @XmlType.propOrder
 this problem is related to the following location:
  at protected java.lang.String org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest.runSafeMode
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest
  at public org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory.createJobScheduleRequest()
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory
  at protected org.w3c.dom.Element org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty.any
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty.customProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto.exportManifestProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto.exportManifestEntity
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto
Property gatheringMetrics is present but not specified in @XmlType.propOrder
 this problem is related to the following location:
  at protected java.lang.String org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest.gatheringMetrics
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest
  at public org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory.createJobScheduleRequest()
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory
  at protected org.w3c.dom.Element org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty.any
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty.customProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto.exportManifestProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto.exportManifestEntity
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto
Property logLevel is present but not specified in @XmlType.propOrder
 this problem is related to the following location:
  at protected java.lang.String org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest.logLevel
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest
  at public org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory.createJobScheduleRequest()
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory
  at protected org.w3c.dom.Element org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty.any
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty.customProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto.exportManifestProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto.exportManifestEntity
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto

```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.importexport.exportManifest.ExportManifestTest.testUnMarshal</b> (click to expand)</summary><p>

```
Could not marshal to XML to string com.sun.xml.bind.v2.runtime.IllegalAnnotationsException: 3 counts of IllegalAnnotationExceptions
Property runSafeMode is present but not specified in @XmlType.propOrder
 this problem is related to the following location:
  at protected java.lang.String org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest.runSafeMode
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest
  at public org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory.createJobScheduleRequest()
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory
  at protected org.w3c.dom.Element org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty.any
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty.customProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto.exportManifestProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto.exportManifestEntity
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto
Property gatheringMetrics is present but not specified in @XmlType.propOrder
 this problem is related to the following location:
  at protected java.lang.String org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest.gatheringMetrics
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest
  at public org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory.createJobScheduleRequest()
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory
  at protected org.w3c.dom.Element org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty.any
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty.customProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto.exportManifestProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto.exportManifestEntity
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto
Property logLevel is present but not specified in @XmlType.propOrder
 this problem is related to the following location:
  at protected java.lang.String org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest.logLevel
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest
  at public org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.JobScheduleRequest org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory.createJobScheduleRequest()
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ObjectFactory
  at protected org.w3c.dom.Element org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty.any
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.CustomProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty.customProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestProperty
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto.exportManifestProperty
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestEntityDto
  at protected java.util.List org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto.exportManifestEntity
  at org.pentaho.platform.plugin.services.importexport.exportManifest.bindings.ExportManifestDto

```

</p></details>

  
Tests run: **1693**, Failures: **5**, Skipped: **1** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5380/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: extensions/src/main/java/org/pentaho/platform/web/http/api/resources/JobScheduleRequest.java [hunk 1]
```diff
@@ -101,6 +101,10 @@ public class JobScheduleRequest implements Serializable {
 
   String timeZone;
 
+  protected String runSafeMode;
```
Comment (ddiroma) [inline extensions/src/main/java/org/pentaho/platform/web/http/api/resources/JobScheduleRequest.java:104]: @srallapa Please update the copyright header to 2023 for this file and all others.

File: extensions/src/main/java/org/pentaho/platform/web/http/api/resources/JobScheduleRequest.java [hunk 2]
```diff
@@ -101,6 +101,10 @@ public class JobScheduleRequest implements Serializable {
...
+  protected String gatheringMetrics;
```
Comment (ddiroma) [inline extensions/src/main/java/org/pentaho/platform/web/http/api/resources/JobScheduleRequest.java:105]: @srallapa You have some checkstyle issues. Please run our checkstyle in Intellij for all .java files using these instructions:
the README.md file in https://github.com/pentaho/pentaho-coding-standards

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1h 33m 29s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl extensions
```
---
:ok_hand: All tests passed!
 
Tests run: **1693**, Failures: **0**, Skipped: **1** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5380/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: extensions/src/main/java/org/pentaho/platform/web/http/api/resources/JobScheduleRequest.java (hunk 1 above)
Comment (srallapa) [reply extensions/src/main/java/org/pentaho/platform/web/http/api/resources/JobScheduleRequest.java:104]: Updated.

File: extensions/src/main/java/org/pentaho/platform/web/http/api/resources/JobScheduleRequest.java (hunk 2 above)
Comment (srallapa) [reply extensions/src/main/java/org/pentaho/platform/web/http/api/resources/JobScheduleRequest.java:105]: Did changes based on checkstyle

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1h 35m 22s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl extensions
```
---
:ok_hand: All tests passed!
 
Tests run: **1693**, Failures: **0**, Skipped: **1** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5380/4/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1h 31m 21s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl extensions
```
---
:ok_hand: All tests passed!
 
Tests run: **1693**, Failures: **0**, Skipped: **1** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5380/5/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by dcleao — APPROVED** [review]

**Review by peterrinehart — APPROVED** [review]

**Pull request merged**